src/module.c

```diff
@@ -7098,14 +7098,14 @@ int RM_Fork(RedisModuleForkDoneHandler cb, void *user_data) {
 /* The module is advised to call this function from the fork child once in a while,
  * so that it can report COW memory to the parent which will be reported in INFO */
 void RM_SendChildCOWInfo(void) {
-    sendChildCOWInfo(CHILD_INFO_TYPE_CURRENT_COW_SIZE, "Module fork");
+    sendChildInfo(CHILD_INFO_TYPE_CURRENT_INFO, 0, "Module fork");
```

Comment:
As I mentioned, I would like to rename this API and let it join that progress party.. 
One option could be to send actual progress on the pipe. And have the fork child sample the total key count instead of the parent. 
In this case, we lose the new metrics that show number of keys and will be only left with a progress percentage metric. (with documentation that says its not percentage of time or size, but rather key count percentage). 
But it would mean it is generic enough to let other forks show progress, regardless of what they do.